fusor-ember-cli/package.json

```diff
@@ -20,7 +20,7 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.1.2",
-    "ember-cli": "1.13.8",
+    "ember-cli": "1.13.12",
```

Comment:
+1, I'm in support of this. Of note, I have committed some code that seemed to pass my linters, but not other people's. I'm not sure if this was due to ember-cli version differences, but it's worth keeping an eye out for.

Also, be sure to uninstall and explicitly reinstall ember-cli globally -- `npm uninstall -g ember-cli && npm install -g ember-cli@1.13.12` in addition to removing the local `node_modules` and reinstalling based on this updated package.json -- `npm install`. The latter is required, but you might as well upgrade globally too.